src/chainparamsbase.cpp

```diff
@@ -9,68 +9,33 @@
 
 #include <assert.h>
 
-#include <boost/assign/list_of.hpp>
-
-using namespace boost::assign;
-
-/**
- * Main network
- */
-class CBaseMainParams : public CBaseChainParams
-{
-public:
-    CBaseMainParams()
-    {
-        networkID = CBaseChainParams::MAIN;
-        nRPCPort = 8332;
-        strDataDir = "alphamain";
-    }
-};
-static CBaseMainParams mainParams;
-
 /**
- * Testnet (v3)
+ * Alpha (pegged to Testnet3)
  */
-class CBaseTestNetParams : public CBaseMainParams
+class CBaseAlphaParams : public CBaseChainParams
 {
 public:
-    CBaseTestNetParams()
+    CBaseAlphaParams()
     {
-        networkID = CBaseChainParams::TESTNET;
         nRPCPort = 4241;
-        strDataDir = "alphatestnet3";
+        strDataDir = "alpha";
```

Comment:
I proposed this change at the very beginning, even without the rest of the changes (in fact when I proposed it it was testnet3, but it was decided to change it to alphatestnet3 instead of alpha. It wouldn't have been disruptive at that point, but I guess alphatestnet3 it is...Now testnet3 is not the only xhain with a datadir different from its bip70 chain name: alpha is flawed that way too (https://github.com/bitcoin/bitcoin/pull/6229 ). Unfortunately by the time this has been read, it is too late.